e2e/utils/TestUtils.ts

```diff
@@ -59,9 +59,7 @@ async function createWindowsUserLegacyProfile(userProfile: RecursivePartial<Sett
   const workdir = await fs.promises.mkdtemp(path.join(os.tmpdir(), 'rd-test-profiles'));
 
   try {
-    for (const packet of [['defaults', userProfile], ['locked', lockedFields]]) {
-      const [registryType, settings] = packet;
-
+    for (const [registryType, settings] of [['defaults', userProfile], ['locked', lockedFields]]) {
```

Comment:
I guess it doesn't matter because registry paths are not case sensitive, but in other places we use `Defaults` and `Locked` for this (`HKCU\\SOFTWARE\\Rancher Desktop\\Profile\\${ registryType }`). But maybe `rdctl create-profile` requires the `--type` value to be all lowercase. Shows that we use the same value for 2 different purposes, and the value is just accidentally compatible with both use cases. But good enough for a test case, I guess.

But I also find the name `registryType` confusing. Shouldn't it be `profileType`?

Reply:
Yeah, it's confusing, but I'm just keeping the same name as before (I find _not_ having the `packet` intermediate value to be clearer).